tests/e2e/tests/content-releases/releases-page.spec.ts

```diff
@@ -43,57 +43,65 @@ describeOnCondition(edition === 'EE')('Releases page', () => {
     await expect(page.getByRole('link', { name: `${newReleaseName}` })).toBeVisible();
   });
 
-  test.fixme(
-    'A user should be able to create a release with scheduling info and view their pending and done releases',
-    async ({ page }) => {
-      // Navigate to the releases page
-      await page.getByRole('link', { name: 'Releases' }).click();
+  test('A user should be able to create a release with scheduling info and view their pending and done releases', async ({
+    page,
+  }) => {
+    // Navigate to the releases page
+    await page.getByRole('link', { name: 'Releases' }).click();
```

Comment:
I recommend using our clickAndWait util, it makes sure to wait that the page is loaded before proceeding, which helps with flakiness